src/openrct2/cheats.c

```diff
@@ -420,6 +422,58 @@ static void cheat_set_staff_speed(uint8 value)
 	}
 }
 
+static void cheat_own_all_land() {
+	uint8 prev_sandbox, prev_no_money, i;
+	uint16 spriteIndex, nextSpriteIndex, spawns_x[2];
+	int min, max, flags, cmd, type;
+	rct_peep *peep;
+
+	//Store states of sandbox, no money, and spawn points to restore when finished
+	prev_sandbox = gCheatsSandboxMode;
+	prev_no_money = gParkFlags & PARK_FLAGS_NO_MONEY;
+	for (i = 0; i < 2; i++) {
+		spawns_x[i] = gPeepSpawns[i].x;
+	}
+
+	//Sandbox mode is needed for this. Also a cheat should not cost, so enable no money.
+	gCheatsSandboxMode = true;
+	if (!prev_no_money)
+		gParkFlags |= PARK_FLAGS_NO_MONEY;
+
+	//Own the entire map
+	min = 32;
+	max = gMapSizeUnits - 32;
+	type = 2;
+	flags = GAME_COMMAND_FLAG_APPLY;
+	cmd = GAME_COMMAND_SET_LAND_OWNERSHIP;
+	game_command_set_land_ownership(&min, &flags, &min, &type, &cmd, &max, &max);
+
+	//remove path from map edge to park entrances
+	editor_check_park_entrance_paths();
```

Comment:
What's the point of calling this function when you don't check its result and the call doesn't alter any state?